scripts/scripts.js

```diff
@@ -293,8 +334,9 @@ async function loadLazy(doc) {
   const element = hash ? main.querySelector(hash) : false;
   if (hash && element) element.scrollIntoView();
 
-  loadHeader(doc.querySelector('header'));
-  loadFooter(doc.querySelector('footer'));
+  await loadHeader(doc.querySelector('header'));
+  await loadFooter(doc.querySelector('footer'));
```

Comment:
I think we should not block here, maybe we should include languageSwitch() in the loadHeader ?

Reply:
Moved the languageSwitch to the header, which makes sense